std/experimental/logger/core.d

```diff
@@ -1618,7 +1618,11 @@ abstract class Logger
 
 private __gshared Logger stdSharedDefaultLogger;
 private shared Logger stdSharedLogger;
-private shared LogLevel stdLoggerGlobalLogLevel = LogLevel.all;
+version (StdUnittest)
+    private enum _defaultLogLevel = LogLevel.all;
```

Comment:
That's a hack not a fix, and it misses quite a few places.

Just look at line 1640, the LogLevel is still *all*

Reply:
It sounds more like the `_defaultLogLevel` isn't properly being used throughout the code base where it should be. I should be able to set it here once and be done, not start combing through all the files to see if there are other "default levels"

Reply:
The global log level is still used for limiting the logger, see https://github.com/dlang/phobos/blob/564dd52a124e04eb7e8706f9b7156ad95d7fa894/std/experimental/logger/core.d#L94-L99 . So this is not a hack, it works fine. In fact, I'd suggest this is exactly the fix that should be done -- if you set the global log level to all, now you get everything as expected.

It would be a further WTF if you had to configure both the global log level and the default logger to see output.

Reply:
Actually, the better option is to have the default logger set to `warning`, and leave the global log level at `all`. If someone generates their own logger for something, then the global logger would need to be adjusted, and possibly the default logger suppressed.

Reply:
My comment was about to things.

1. The version statement makes it so that the unit tests actually do not test the default behavior of the module. IMO that is not way to make the tests pass.
2. @schveiguy proposal to set the LogLevel of the default logger to warning is a better solution than setting the globalLogLevel to warning.